src/Domain/Creators/TimeDependence/Composition.tpp

```diff
@@ -0,0 +1,146 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Domain/Creators/TimeDependence/Composition.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain {
+namespace creators {
+namespace time_dependence {
+namespace detail {
+template <typename... MapsSoFar>
+auto combine_coord_maps(
+    CoordinateMap<Frame::Grid, Frame::Inertial, MapsSoFar...>
```

Comment:
It seems like these are copying the maps a lot. They're probably pretty small most of the time, but it would still be better to move or reference wherever possible.